circle.yml

```diff
@@ -70,7 +78,9 @@ workflows:
           filters:
             tags:
               only: /^v.*/
-
+      - nuke_phx_devops:
+          requires:
+            - test
```

Comment:
This will run aws-nuke after every commit... to delete resources more than 48h old. What's the purpose of that? I don't think we need this here; we should *only* have the `nuke_phx_devops` run once nightly.